src/components/Navigation/DesktopNavigation/DropdownElement.jsx

```diff
@@ -0,0 +1,26 @@
+import { NAV_ELEMENT_TYPES } from '../../../constants/navigation';
+import classNames from 'classnames/bind';
+import styles from './DropdownElement.module.css';
+
+const cx = classNames.bind(styles);
+
+const DropdownElement = ({ type, name, icon }) => (
+    <li>
+        {type === NAV_ELEMENT_TYPES.SUBHEADER ? (
+            <div className={styles.navbarItem}>{name}</div>
+        ) : (
+            <a
+                className={cx({
+                    navbarItem: true,
```

Comment:
navbarItem: true, are you sure we can't replace it with navbarItem only

Reply:
We cannot use shorter notation for navbarItem property as we haven't defined it as separate variable anywhere else